ClientApp/src/modules/authorization/views/CreateUserView/CreateUserView.tsx

```diff
@@ -58,9 +87,16 @@ const CreateUserView: React.FC = ({children}) => {
                     placeholder="confirm password"
                     style={input}
                 />
+                <Input value={telephone} onChange={onChangeTelephone} type="tel" placeholder="500500500" style={input} />
+
+                <Box style={boxswt} mb="0">
+                    <Switch onChange={()=>toggleType} id="type" style={swt}/>
```

Comment:
Function that returns function xd. Just pass function
```suggestion
                    <Switch onChange={toggleType} id="type" style={swt}/>
```

Reply:
is it working without binding?

Reply:
Yep, its exactly the same concept